### PR TITLE
configFile delete logs

### PR DESCRIPTION
**Pull request by isha309**
- FFDC improvement needed as discussed in defect https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=583147

Tested By :
```
curl -k -H "X-Auth-Token: $bmc_token" -X DELETE [https://${bmc}/ibm/v1/Host/ConfigFiles/<filename>

curl -k -H "X-Auth-Token: $bmc_tokens" -X POST https://${bmc}/ibm/v1/Host/ConfigFiles/Actions/IBMConfigFiles.DeleteAll
```

### CODE REVIEW

File: include/ibm/management_console_rest.hpp [hunk 1]
```diff
@@ -485,13 +491,13 @@ inline void
 {
     std::string filePath("/var/lib/bmcweb/ibm-management-console/configfiles/" +
                          fileID);
-    BMCWEB_LOG_DEBUG << "Removing the file : " << filePath << "\n";
+    BMCWEB_LOG_CRITICAL << "INFO:Removing the file : " << filePath << "\n";
```
Comment (sunharis) [inline include/ibm/management_console_rest.hpp:494]: This log is not needed to be in the journal. Please revert

File: include/ibm/management_console_rest.hpp [hunk 2]
```diff
@@ -420,10 +420,16 @@ inline void
             asyncResp->res.result(
                 boost::beast::http::status::internal_server_error);
             asyncResp->res.jsonValue["Description"] = internalServerError;
-            BMCWEB_LOG_DEBUG << "deleteConfigFiles: Failed to delete the "
+            BMCWEB_LOG_ERROR << "deleteConfigFiles: Failed to delete the "
                                 "config files directory. ec : "
                              << ec;
         }
+        else
```
Comment (sunharis) [inline include/ibm/management_console_rest.hpp:427]: I dont think else block is needed.

File: include/ibm/management_console_rest.hpp [hunk 3]
```diff
@@ -485,13 +491,13 @@ inline void
...
                          fileID);
-    BMCWEB_LOG_DEBUG << "Removing the file : " << filePath << "\n";
+    BMCWEB_LOG_CRITICAL << "INFO:Removing the file : " << filePath << "\n";
     std::ifstream fileOpen(filePath.c_str());
     if (static_cast<bool>(fileOpen))
     {
         if (remove(filePath.c_str()) == 0)
         {
-            BMCWEB_LOG_DEBUG << "File removed!\n";
+            BMCWEB_LOG_CRITICAL << "INFO:configFile removed!\n";
```
Comment (sunharis) [inline include/ibm/management_console_rest.hpp:500]: print the file path which was removed

**Review by sunharis — CHANGES_REQUESTED** [review]

**Comment by sunharis** [comment]
Tested by ?

File: include/ibm/management_console_rest.hpp (hunk 1 above)
Comment (isha309) [reply include/ibm/management_console_rest.hpp:494]: @sunharis 
if I remove this log , path should be added in logs at 500 , 508 ,  516 separately 
500 BMCWEB_LOG_CRITICAL << "INFO:configFile removed!\n";
508 BMCWEB_LOG_ERROR << "File not removed!\n";
516 BMCWEB_LOG_ERROR << "File not found!\n";

File: include/ibm/management_console_rest.hpp (hunk 3 above)
Comment (isha309) [reply include/ibm/management_console_rest.hpp:500]: path can be checked from 494 line

File: include/ibm/management_console_rest.hpp (hunk 1 above)
Comment (sunharis) [reply include/ibm/management_console_rest.hpp:494]: They are already error logs which will come in the journal. You can add the file path there to make them meaningful.

**Comment by isha309** [comment]
> Tested by ?

added tested by section

File: include/ibm/management_console_rest.hpp [hunk 4]
```diff
@@ -420,10 +420,12 @@ inline void
             asyncResp->res.result(
                 boost::beast::http::status::internal_server_error);
             asyncResp->res.jsonValue["Description"] = internalServerError;
-            BMCWEB_LOG_DEBUG << "deleteConfigFiles: Failed to delete the "
+            BMCWEB_LOG_ERROR << "deleteConfigFiles: Failed to delete the "
                                 "config files directory. ec : "
                              << ec;
         }
+        BMCWEB_LOG_CRITICAL
+            << "INFO:config files directory delete successful. PATH: " << loc;
```
Comment (gtmills) [inline include/ibm/management_console_rest.hpp:428]: How much does this log?

**Comment by gtmills** [comment]
Can some of this go upstream?

File: include/ibm/management_console_rest.hpp (hunk 4 above)
Comment (isha309) [reply include/ibm/management_console_rest.hpp:428]: I am not sure if I  understand the question , 
if you are asking how many times we will get this log --> every time deletion of configfile is successsful we will get this log .

**Comment by isha309** [comment]
> Can some of this go upstream?
we can do code it upstream , but I am not sure if its needed . 
we need these log to debug the above defect scenario .

**Review by sunharis — APPROVED** [review]

**Comment by gtmills** [comment]
> we can do code it upstream , but I am not sure if its needed .
> we need these log to debug the above defect scenario .

So do we need this for 1060? Do we need this for Pnext? 
Is this something we will remove later in 1060?

**Pull request merged**